src/readbin/symbols.ml

```diff
@@ -66,4 +66,21 @@ let read ?demangle ~filename arch base  : string table =
             let mem = Memory.view ~from ~words base |> ok_exn in
             let name = demangle name in
             Table.add syms mem name |> ok_exn
-          with exn -> syms))
+          with _exn -> syms))
+
+let read_addrset filename : Addr.Set.t =
+  Addr.Set.t_of_sexp @@ Sexp.load_sexp filename
+
+let write_addrset ?filename (addr_set : Addr.Set.t) : unit =
+  match filename with
+  | None -> Sexp.output Out_channel.stdout @@ Addr.Set.sexp_of_t addr_set
```

Comment:
There is no need to prefix `stdout` with `Out_channel`. Try not to clobber your code with extra code, that doesn't add any more information or helps readability.